public/app/plugins/datasource/influxdb/datasource_backend_mode.test.ts

```diff
@@ -140,4 +141,74 @@ describe('InfluxDataSource Backend Mode', () => {
       expect(fetchReq.queries[0].tags?.[1].value).toBe(adhocFilters[0].value);
     });
   });
+
+  // ignore those tests for now
```

Comment:
I'd rather not merge anything commented out, can we push this to another PR?

Reply:
You are right those are simply unnecessary. I added them for this PR (https://github.com/grafana/grafana/pull/75653) but you are right. I will delete them.